repos/fdbt-site/tests/pages/api/productDateInformation.test.ts

```diff
@@ -152,7 +152,7 @@ describe('productDataInformation', () => {
 
         expect(updateSessionAttributeSpy).toBeCalledWith(req, PRODUCT_DATE_ATTRIBUTE, {
             startDate: '2020-12-12T00:00:00.000Z',
-            endDate: '2020-12-15T23:59:59.999Z',
+            endDate: '2020-12-15T23:59:59.000Z',
             dateInput: {
```

Comment:
What does changing from 999 to 000 achieve?

Reply:
So we've changed the time we assign to end dates to only go up to seconds, and not to include the 999 milliseconds - therefore we expect this new time to be the value that's used. That make sense?

Reply:
Yes, thanks :)